operator/contractpkg/main.go

```diff
@@ -129,7 +135,11 @@ func (m *ContractMap) process(name string, b *ContractMapBuild) error {
 		for libName, link := range b.LinkedLibs {
 			link = strings.ReplaceAll(link, "_", "-")
 			requiredBuilds = append(requiredBuilds, link)
-			linkedContracts[libName] = fmt.Sprintf(`{{index .status.resolvedContractAddresses "%s"}}`, link)
+			l := fmt.Sprintf(`{{index .status.resolvedContractAddresses "%s"}}`, link)
+			if helmCompatible {
+				l = fmt.Sprintf("{{`%s`}}", l)
```

Comment:
TODO: This should take place in the Makefile (when copying the contracts)

Reply:
Why do we need the non-helm compatible version here? Surely if this is valid YAML, then let's just do this version